### DISCUSSION TITLE
Support for new major versions of FastAPI and Pydantic

### DISCUSSION THREAD

**Discussion by bkis** [fastapi-users/fastapi-users#1244] [General]
With [Pydantic v2](https://docs.pydantic.dev/latest/blog/pydantic-v2-final/) being released now and a [new FastAPI release with Pydantic v2 support coming very soon](https://github.com/tiangolo/fastapi/releases/tag/0.99.0), what is the state of FastAPI-Users in terms of support for those new releases? I don't want to push this or anything, just asking out of curiosity. Have you played around with the recent FastAPI betas already? Thank you for all your work!

**Comment by frankie567** [comment]
Yes, definitely want to support Pydantic V2 and FastAPI. I don't have much time right now to tackle this, but definitely in the plans!

**Comment by frankie567** [comment]
Released in [v12.1.0](https://github.com/fastapi-users/fastapi-users/releases/tag/v12.1.0) 🎉

**Comment by peter65374** [comment]
> Released in [v12.1.0](https://github.com/fastapi-users/fastapi-users/releases/tag/v12.1.0) 🎉



> Released in [v12.1.0](https://github.com/fastapi-users/fastapi-users/releases/tag/v12.1.0) 🎉



> Released in [v12.1.0](https://github.com/fastapi-users/fastapi-users/releases/tag/v12.1.0) 🎉

That's cool!!! I am wondering if fastapi-user supports pydantic v2 and when I could upgrade my FastAPI server to 1.000. respect to you.
